source/module_hamilt_general/module_xc/xc_functional_gradcorr.cpp

```diff
@@ -89,6 +91,10 @@ void XC_Functional::gradcorr(double &etxc, double &vtxc, ModuleBase::matrix &v,
 	gdr1 = new ModuleBase::Vector3<double>[rhopw->nrxx];
 	if(!is_stress)	h1 = new ModuleBase::Vector3<double>[rhopw->nrxx];
 	
+	size_t record_num = sizeof(ModuleBase::Vector3<double>)*rhopw->nrxx + sizeof(std::complex<double>)*rhopw->npw + sizeof(double)*rhopw->nrxx;
+	if(!is_stress) record_num += sizeof(ModuleBase::Vector3<double>)*rhopw->nrxx;
+	ModuleBase::Memory::record("XC_Functional::gradcorr", record_num);
```

Comment:
The function will be called many times, but the memory will not accumulate, so I think it seems inappropriate to record the memory here.